blogposts/2021/release-3.34.md

```diff
@@ -0,0 +1,45 @@
+    url: https://github.com/sourcegraph/sourcegraph/issues/27058
+    category: Search
+  - description: 'To help site admins debug issues with webhook delivery, we have added logging of incoming Batch Changes webhooks. By default, sites without encryption will log webhooks for three days and sites with encryption will not log webhooks.'
+    url: https://docs.sourcegraph.com/admin/config/batch_changes#incoming-webhooks
+    category: Admin
+  - description: 'To allow writing to the `.github` directory in repositories, Batch Changes now requests the `workflow` scope on GitHub personal access tokens. If you have already configured a GitHub PAT for Batch Changes, we suggest adding this scope.'
+    url: https://docs.sourcegraph.com/batch_changes/how-tos/configuring_credentials
+    category: Batch Changes
+  - description: "To increase search speed, the search UI's repository count and the GraphQL API's `search().repositories`/`search().repositoriesCount` have changed semantics from being the set of searchable repositories to the set of repositories with matches. In a future release, we’ll introduce separate fields for searchable repositories."
+    url: https://github.com/sourcegraph/sourcegraph/issues/26995
+    category: Search
+  - description: 'Version contexts and `repogroup` filters (deprecated in v3.33) have been removed in favor of search contexts.'
+    url: https://about.sourcegraph.com/blog/introducing-search-contexts/
+    category: Search
+  - description: 'Saved search notifications (deprecated in v3.31.0) have been removed in favor of code monitoring.'
+    url: https://docs.sourcegraph.com/code_monitoring
+    category: Search
+---
+
+Sourcegraph 3.34 is now available! Here are some highlights from this release:
+
+## Troubleshoot your searches with the "no results" page
+
+Searches that return no results can result in a confusing experience. When this occurs, we now display helpful tips on how to use search, examples of how to solve common query issues, and links to tutorial videos addressing common use cases.
+<img class="blog-image" alt="Find implementations for Go" src="https://storage.googleapis.com/sourcegraph-assets/blog/3.34/no_results_page.png">
+
+## Quickly navigate your Go code with Find Implementations
+
+If you have precise code intelligence enabled for your Go repositories, you can now click on "Find Implementations" to navigate to a symbol's interface definition. If you're at the interface definition itself, clicking on the same button will show all the places where the interface is being implemented, allowing you to explore how it's being used by other users across repositories. It can also show you which interfaces a struct implements.
+<img class="blog-image" alt="Find implementations for Go" src="https://user-images.githubusercontent.com/1657213/142938393-7aed0c41-28b8-4cab-bf0d-2f9fd7f1078c.png">
```

Comment:
@macraig these images are showing very nearly the exact same thing as best I can tell, what's the difference between the two?

Reply:
@iskyOS I wouldn't block merging the release post on this, a fix could come in a follow up (or not).

Reply:
@jyavorska fair point. One shows where an interface is implemented across many different repositories, the other shows how you see where the interface is defined from a repo that implements it. I understand this seems too similar and confusing (and we're actually currently looking into a way of making these two directions clearer in the UI). I think we can remove the second image, I'll do that right now.